packages/shared/src/components/streak/popup/ReadingStreakPopup.tsx

```diff
@@ -83,7 +86,14 @@ export function ReadingStreakPopup({
         <StreakSection streak={streak.current} label="Current streak" />
         <StreakSection streak={streak.max} label="Longest streak 🏆" />
       </div>
-      <div className="mt-6 flex flex-row gap-2">{streaks}</div>
+      <div
+        className={classNames(
+          'mt-6 flex flex-row gap-2',
+          fullWidth && 'justify-between',
```

Comment:
We could've maybe used a className object prop here, but I really don't have a strong opinion here. Just makes it more flexible. Not necessary, this is optional.

Reply:
since it's a classname specific to the streak days list, we would have to create a `className` prop which allows passing classnames to each section of the component internals. My suggestion would be to only do that if we need to control this component's UI further in the future 🙏